experiments/pagination/README.md

```diff
@@ -0,0 +1,98 @@
+# Use of Paging for large sets
+
+This experiment demonstrates why pagination might provide a better experience, and allows hands-on, immediate feedback
+on pagination fields.
+
+## Problem
+At some point, the optimal query has been created, using all the performance enhancements, but still produces a lot
+of hits and takes a long time to complete. Such a delay can be a bad experience for the user and uses a lot of 
+system resources. 
+
+## Goals
+This experiment tries to show the power of Pagination and the use of [Querybuilder](https://helpx.adobe.com/experience-manager/6-5/sites/developing/using/reference-materials/javadoc/com/day/cq/search/QueryBuilder.html)
+properties.  A customer’s search may include introduction of their own custom APIs, and not have the UI call
+Querybuilder directly.  That is fine, as this experiment simply tries to illustrate the principles of pagination. 
+
+## Non-Goals
+It does not try to teach how to:
+ * create a beautiful web page
+ * script perfect JS/JQuery
+ * optimize a search for best performance
+ * validate field values
+ * etc.
```

Comment:
@top I think bullets (both Goals and Non-Goals) should probably start with a capital letter.

Reply:
From https://www.editage.com/insights/scientific-writing-tips-on-capitalization-in-bullet-point-lists#:~:text=If%20a%20list%20is%20introduced,begin%20with%20a%20capital%20letter.&text=Never%20use%20a%20capital%20letter,stop%2C%20thereby%20completing%20the%20sentence.

`When a list is introduced by an incomplete sentence, as in "Three main types of vegetative parts used for propagation are," leave the incomplete introduction "open" (no punctuation). Never use a capital letter to begin an item in such lists; instead, begin with a lowercase letter and end the last item with a full stop, thereby completing the sentence.`